screens/Search/index.tsx

```diff
@@ -12,32 +12,48 @@ import CustomText from "@/components/ui/CustomText"
 import { User } from "@/types"
 import { theme } from "@/theme"
 import CheckIcon from "@/components/vectors/CheckIcon"
+import api from "@/service"
 
 const FindFriends = () => {
   const navigation = useNavigation()
   const [search, setSearch] = useState("")
-  const [addedFriends, setAddedFriends] = useState<User[]>([])
-  const [filteredFriends, setFilteredFriends] = useState(availableFriends)
+  const [addedUsers, setAddedUsers] = useState<User[]>([])
+  const [filteredUsers, setFilteredUsers] = useState(availableFriends)
+  const [allUsers, setAllUsers] = useState<User[]>([])
+  useEffect(() => {
+    fetchUsers()
+  }, [])
+  const fetchUsers = async () => {
+    try {
+      const response = await api.get(`/api/users/`)
+      const users = response.data.map((user: any) => ({
+        id: user.id,
+        name: user.name,
+        email: user.email,
+        imageUrl: user.profile || "",
+      }))
+      setAllUsers(users)
+      setFilteredUsers(users)
+    } catch (error) {
+      console.error("error fetching users", error)
+    }
+  }
```

Comment:
can you order this and start by the function creation and call it in a useEffect later